### PR TITLE
Feature/upload media scripts

### PR DESCRIPTION
**Pull request by gpstrange**
Upload media scripts for fact-check and fearspeech datasets

### CODE REVIEW

**Comment by gpstrange** [comment]
Add access-token, development.env file to run factcheck script.
Update fear_speech_data.json with fearspeech data to run fearspeech script.

File: src/scripts-upload-media/factcheck.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const knownMediaTypes = ['text', 'image', 'video'];
+const ignoreMediaTypes = ['tweet', 'facebook', 'instagram'];
+const failedRequestsFile = "factcheck-failed-requests.json";
+
+const insertPosts = async () => {
+    const { connect, get, bulkWrite } = require("../backend/core/mongo/index")
+    await connect()
+
+    const stories = await get(
+        "kosh_metadata",
```
Comment (dennyabrain) [inline src/scripts-upload-media/factcheck.js:20]: Lets make this a const variable at the top of the file. We can name it "FACTCHECK_DB_NAME". I believe this is not the right name of that collection in our production database. so having it at the top will make it easy to change later

File: src/scripts-upload-media/factcheck.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            if (post.e_kosh_id) continue
+            if (!knownMediaTypes.includes(post.mediaType)) {
+                unknownMediaTypes.add(post.mediaType)
+                continue
+            }
+            if (post.mediaType === "text" && !post.s3URL) {
+                const fileId = uuid();
+                const { uploadData } = require("./s3");
+                const awsResponse = await uploadData(post.content, fileId);
+                post.s3URL = awsResponse.Location
```
Comment (dennyabrain) [inline src/scripts-upload-media/factcheck.js:42]: replace this with post.s3uRL = `https://fs.tattle.co.in/service/kosh/file/${fileId}`
We'd not want to expose the raw s3 url anywhere on the website. I should have flagged this earlier. Sorry about that.

File: src/scripts-upload-media/fearspeech.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const datasource = "efb05069-e220-4b82-8849-98d14bffdafd";
+
+const failedRequestsFile = "fear-speech-failed-requests.json";
+const insertPost = async () => {
+    const posts = []
+    for (const post of Object.values(stories)) {
+        const fileId = uuid();
+        const { uploadData } = require("./s3");
+        const awsResponse = await uploadData(post.message_text, fileId);
+        post.s3URL = awsResponse.Location
```
Comment (dennyabrain) [inline src/scripts-upload-media/fearspeech.js:20]: same feedback. Replace this with post.s3uRL = https://fs.tattle.co.in/service/kosh/file/${fileId}

**Pull request merged**